core/src/main/scala/com/nvidia/spark/rapids/tool/qualification/QualOutputWriter.scala

```diff
@@ -1020,6 +1057,50 @@ object QualOutputWriter {
     }.flatten.toSet
   }
 
+  private def constructAllOperatorsInfo(
```

Comment:
Something bugs me about the operator files.
Is it possible to make this file at least sorted.

- if we sort by (count, sql, operatorName, supported), then we will be able to see the to operators at the top but at the same time it will be noisy when we start to count expressions per exec. Because for sure, expressions will be more count than execs.
- if we sort by (sql, count..) then the report will be more readable as SQLs appear in order.

Reply:
Thanks for the suggestion @amahussein .  I have yet to address this as this might cause memory/runtime overhead. Thinking of a better way to update this.